ext/native-decls/BreakOffVehicleWheel.md

```diff
@@ -0,0 +1,31 @@
+---
+ns: CFX
+apiset: client
+---
+## BREAK_OFF_VEHICLE_WHEEL
+
+```c
+void BREAK_OFF_VEHICLE_WHEEL(Vehicle vehicle, int wheelIndex, BOOL leaveDebrisTrail, BOOL deleteWheel, BOOL unknownFlag, BOOL putOnFire);
+```
+
+Break off vehicle wheel by index. The `leaveDebrisTrail` flag requires `putOnFire` to be true.
+
+## Examples
```

Comment:
Does it matter if the example is before or after paramaters or does hugo automatically order this

Reply:
Hmmm, good note. I used my old `GET_INTERIOR_ROOM_NAME` native as a reference, but it seems on native reference the `Parameters` block is not visible. I guess the order is important indeed. Would also need to fix old natives then.